src/sdk/cpp/lib/src/iotea.cpp

```diff
@@ -200,27 +195,24 @@ Event Event::FromJson(const json& j) {
     auto return_topic = j.contains("returnTopic") ? j["returnTopic"].get<std::string>() : "";
     auto when_ms = j["whenMs"].get<int64_t>();
 
-    timepoint_t when{std::chrono::milliseconds{when_ms}};
-    return Event{subject, feature, value, type, instance, return_topic, when};
+    return Event{subject, feature, value, type, instance, return_topic, when_ms};
 }
 
 OutgoingCall::OutgoingCall(const std::string& talent_id, const std::string& channel_id, const std::string& call_id,
                            const std::string& func, const json& args, const std::string& subject,
-                           const std::string& type)
+                           const std::string& type, int64_t when)
     : talent_id_{talent_id}
     , channel_id_{channel_id}
     , call_id_{call_id}
     , func_{func}
-    , args_{args}
+    , args_(args)
```

Comment:
What is the reason to use this type of bracket "(" here?

Reply:
If nlohman::json values are initialized with curly braces they get wrapped in an array with we don't want in this case.